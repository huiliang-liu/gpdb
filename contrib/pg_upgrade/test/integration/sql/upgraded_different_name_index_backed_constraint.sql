SELECT * FROM table_with_unique_constraint;
SELECT * FROM table_with_primary_constraint;
