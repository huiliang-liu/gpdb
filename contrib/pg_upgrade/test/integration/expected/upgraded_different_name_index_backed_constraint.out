SELECT * FROM table_with_unique_constraint;
 author | title 
--------+-------
 1      | 1     
 2      | 2     
(2 rows)
SELECT * FROM table_with_primary_constraint;
 author | title 
--------+-------
 1      | 1     
 2      | 2     
(2 rows)
